src/components/sidebar/user-profile/Contacts.js

```diff
@@ -113,13 +113,14 @@ const Contacts = ({classes, email, skypeUsername, skypeForBusiness, phoneNumber}
 
 Contacts.propTypes = {
   classes: PropTypes.object.isRequired,
-  email: PropTypes.string.isRequired,
+  email: PropTypes.string,
```

Comment:
Why is email not required anymore?

Reply:
Let's imagine, when we invited a new user to an organization and started a conversation with him. Then click to a sidebar, and want to see information about this user.

From out subscriptions events, we will receive info about the user - https://github.com/ubergrape/grape-web-client/blob/master/src/app/subscribe.js#L105 here.
But, without email. So, at first render, a component will not know about user email. That's why email not required anymore.

But, if you clicked on user profile page, you still have no email for this user. In this situation, we will get user info with the email from the server.
https://github.com/ubergrape/grape-web-client/blob/5bb5042df7fe7456a1ee728e768ae0182546f126/src/components/sidebar/user-profile/UserProfile.js#L110

Backend guys don't wanna to include email to a response from `organization.joined` because it's a pretty rare case where the user will click immediately to user profile page in the sidebar after the user was invited to the organization.

And in get_overview request email included. So, if you got a conversation with new user and refresh the page, you will receive this new user from get_overview.

Hope you understand my message:)

What you think about it?

Reply:
Makes sense! Thanks for the clarification 👍